src/lib/components/identity/identity-manager.svelte

```diff
@@ -51,6 +54,12 @@
             color: 'dark',
         },
     ]
+    export let identityFilter: FilterCheckbox[] = [
```

Comment:
I see you only use this filter locally, do you need to export it

Reply:
You are right, removed the export and changed to const